cibuildwheel/util.py

```diff
@@ -34,6 +35,18 @@ def get_build_verbosity_extra_flags(level: int) -> List[str]:
         return []
 
 
+def read_python_configs(input_path: Union[str, os.PathLike]) -> List[Dict[str, str]]:
+    loaded_file = toml.load(input_path)
+    results: List[Dict[str, str]] = []
+
+    for k, v in loaded_file.items():
+        results.append({'identifier': k, **v})
+        if 'cp27-manylinux_' in k:
+            results.append({'identifier': k, 'version': v['version'], 'path_str': f'{v["path_str"]}u'})
```

Comment:
This is because we use TOML instead of YAML, we can't have a list with dicts.

Would json be all that bad?